modules/vaos/config/routes.rb

```diff
@@ -16,6 +16,7 @@
       resources :cancel_reasons, only: :index
       resources :available_appointments, only: :index
       resources :limits, only: :index
+      get 'visits/:schedule_type', to: 'visits#index'
```

Comment:
There's probably a good reason for this but I'm not seeing it. Why not controller_name#index?

Reply:
we need `../:schedule_type` appended, is there another way to do that?

Reply:
Ahh, right I see. If schedule_type is something limited, might want to limit the route to only those constraints, easy to do if constraints are a constant in your controller.

```
# from config/routes.rb
get '/sessions/:type/new',
      to: 'v0/sessions#new',
      constraints: ->(request) { V0::SessionsController::REDIRECT_URLS.include?(request.path_parameters[:type]) }
```

but also not critical if you're doing that in the controller already.

Reply:
I personally think for readability it helps to know how routes are limited

Reply:
alternately just do 

`resources :visits, only: :index`

and just make `:schedule_type` a required parameter

Reply:
If we're going to do anything custom like this for vanity urls, I would suggest introducing routing specs `specs/routing/vaos/visits_spec.rb` or something too.